front/lib/api/tables.ts

```diff
@@ -369,14 +369,14 @@ export async function rowsFromCsv({
 
   let i = 0;
   const parser = parse(csv, { delimiter });
-  const valuesByCol: Record<string, string[]> = {};
+  const valuesByCol: Record<string, string[]> = Object.create(null);
   for await (const anyRecord of parser) {
     if (i++ >= rowIndex) {
       const record = anyRecord as string[];
       for (const [i, h] of header.entries()) {
         try {
           valuesByCol[h] ??= [];
-          (valuesByCol[h] as string[]).push(record[i] || "");
+          (valuesByCol[h] as string[]).push(record[i] ?? "");
```

Comment:
What types of value are we going to get that we didn't get? Are they desirable?

Reply:
this change is preemptive and not related to the fix, here if we get a value of 0 in `record[i]` it gets converted to ""